node/modules/storageminer.go

```diff
@@ -468,6 +469,13 @@ func BasicDealFilter(user dtypes.DealFilter) func(onlineOk dtypes.ConsiderOnline
 				return false, fmt.Sprintf("cannot seal a sector before %s", deal.Proposal.StartEpoch), nil
 			}
 
+			// Reject if it's more than 90 days in the future
+			// TODO: read from cfg
+			maxStartEpoch := ht + abi.ChainEpoch(90*builtin.EpochsInDay)
```

Comment:
90 days is probably way more than most miners would expect.

I'd set this to 7 days for now, than work on getting this to the config